Makefile

```diff
@@ -30,8 +30,12 @@ build:
 		wget https://github.com/Xilinx/RapidWright/releases/download/v2020.2.5-beta/rapidwright_api_lib-2020.2.5-patch1.zip && \
 		unzip -o rapidwright_api_lib-2020.2.5-patch1.zip && \
 		popd
+	pushd third_party/prjoxide/libprjoxide && \
+		( curl --proto '=https' -sSf https://sh.rustup.rs | sh -s -- -y ) && \
+		PATH=${PATH}:${HOME}/.cargo/bin cargo install --path prjoxide --all-features && \
```

Comment:
I saw a failure for which capnp cannot be found. Given that it gets installed with conda we may try adding the `@$(IN_CONDA_ENV)` prefix to the build step.

IIRC, the capnp default package for debian is not up-to-date with the version we need (`0.8.0`)

Reply:
Done

Reply:
My bad sorry, I guess `@$(IN_CONDA_ENV)` should be put just before `pushd`

Reply:
I'm still having some problems, I suspect because I'm overriding `PATH` to include the rust compiler too. Will investigate in a bit.